### ISSUE TITLE
[aes] Consider adding a repetition check for the EDN interface

### ISSUE THREAD

**Issue by vogelpi** [lowRISC/opentitan#15777]
As part of #14240 we were thinking to adding repetition checks for the EDN interface of AES. At the moment there is no repetition check as AES doesn't require high-quality entropy. However, it would make it harder for someone to replicate the masking off analysis which requires all 32-bit LFSRs in the PRNG to be seeded to the same specific seed.

For future versions, we should consider adding this repetition check and connect it to a recoverable/info alert.

**Comment by tjaychen** [comment]
yeah, we could increase the resolution at which edn does the repetition checks? I think it's 64b right now (could be wrong), and we could make it 32b for example.  Which is probably the width that most blocks fetch.

**Comment by vogelpi** [comment]
Backlog sounds right for this. We don't need this for M2.5.
